packages/sandbox/src/constants.ts

```diff
@@ -67,7 +67,36 @@ export const DEFAULT_FILES: SandboxFiles = {
   flex-direction: column;
   gap: 1rem;
   padding: 1rem;
+  background-color: #f1f0ef;
+  height: calc(100vh - var(--gateway-header-height))
```

Comment:
Let's switch `height` here to `min-height` to allow the content to expand past the viewport height on smaller screens (or when more content is added that takes up vertical space).

Reply:
great catch